### PR TITLE
Remove inline styles from Web/API docs

### PR DESCRIPTION
**Pull request by Blakelist7**
Part of fixing #5438

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks very much! Good job. I've added a few review comments. Probably more than needed but I figure we will try be consistent :-)

**Comment by sideshowbarker** [comment]
> Thanks very much! Good job. I've added a few review comments. Probably more than needed but I figure we will try be consistent :-)

@hamishwillee Those comments don’t seem to be showing up here yet?

**Review by hamishwillee — CHANGES_REQUESTED** [review]

**Comment by hamishwillee** [comment]
Thanks @sideshowbarker  - made the review non-pending.

File: files/en-us/web/api/selection/index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@ <h3 id="Multiple_ranges_in_a_selection">Multiple ranges in a selection</h3>
 
 <p>A selection object represents the {{DOMxRef("Range")}}s that the user has selected. Typically, it holds only one range, accessed as follows:</p>
 
-<div style="overflow: hidden;">
+<div>
```
[the remark this answers is not in the document]
Comment (Blakelist7) [reply files/en-us/web/api/selection/index.html]: Sure.

File: files/en-us/web/api/sensor/stop/index.html [hunk 2]
```diff
@@ -25,9 +25,9 @@ <h3 id="Parameters">Parameters</h3>
 
 <p>None.</p>
 
-<p><span
-    style='background-color: #333333; color: #ffffff; font-family: x-locale-heading-primary,zillaslab,Palatino,"Palatino Linotype",x-locale-heading-secondary,serif; font-size: 1.6rem; letter-spacing: -0.00333rem;'>Return
-    value</span></p>
+<h3>
```
[the remark this answers is not in the document]
Comment (Blakelist7) [reply files/en-us/web/api/sensor/stop/index.html]: Sure.

File: files/en-us/web/api/serviceworkerglobalscope/onsync/index.html [hunk 3]
```diff
@@ -6,16 +6,12 @@
...
 <p>The <strong><code>ServiceWorkerGlobalScope.onsync</code></strong> event of the
-    {{domxref("ServiceWorkerGlobalScope")}} interface is<span
-        style="line-height: 1.5;"> fired whenever a </span><code
-        style="font-style: normal; font-weight: normal; line-height: 1.5;"><a href="/en-US/docs/Web/Events/SyncEvent">SyncEvent</a></code><span
-        style="line-height: 1.5;"> event occurs. This is triggered when a call
-        to </span><a href="/en-US/docs/Web/API/SyncManager/register"
-        style="line-height: 1.5;"
-        title="The register method of the SyncManager interface returns a Promise that resolves to a SyncRegistration instance."><code>SyncManager.register</code></a><span
-        style="line-height: 1.5;"> is made from a service worker client page. The attempt
+    {{domxref("ServiceWorkerGlobalScope")}} interface is fired whenever a <code><a href="/en-US/docs/Web/Events/SyncEvent">SyncEvent</a></code>
```
[the remark this answers is not in the document]
Comment (Blakelist7) [reply files/en-us/web/api/serviceworkerglobalscope/onsync/index.html]: Sure.

File: files/en-us/web/api/staticrange/endcontainer/index.html [hunk 4]
```diff
@@ -12,15 +12,13 @@
...
-<p><span style="font-size: 1rem; letter-spacing: -0.00278rem;">{{APIRef("DOM
-    WHATWG")}}</span></p>
+<p>{{APIRef("DOM
+    WHATWG")}}</p>
 
-<p><span class="seoSummary"><span
-      style="font-size: 1rem; letter-spacing: -0.00278rem;">The </span><strong
-      style="font-size: 1rem; letter-spacing: -0.00278rem;"><code>endContainer</code></strong><span
-      style="font-size: 1rem; letter-spacing: -0.00278rem;"> property of the
+<p><span class="seoSummary">The <strong
```
[the remark this answers is not in the document]
Comment (Blakelist7) [reply files/en-us/web/api/staticrange/endcontainer/index.html]: Done.

File: files/en-us/web/api/sensor/stop/index.html [hunk 5]
```diff
@@ -25,9 +25,7 @@ <h3 id="Parameters">Parameters</h3>
 
 <p>None.</p>
 
-<h3>
-  Return value
-</h3>
+<h3>Return value</h3>
```
Comment (hamishwillee) [inline files/en-us/web/api/sensor/stop/index.html:28]: id?

Comment (hamishwillee) [reply files/en-us/web/api/sensor/stop/index.html:28]: This is the only nit left :-)

Comment (Blakelist7) [reply files/en-us/web/api/sensor/stop/index.html:28]: id="Return" or "Return_value"?

Comment (Blakelist7) [reply files/en-us/web/api/sensor/stop/index.html]: Done

**Review by hamishwillee — APPROVED** [review]
Awesome. Thanks so much for helping with this huge (and fiddly) job.

**Pull request merged**